pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala

```diff
@@ -0,0 +1,39 @@
+package worker
+
+import org.apache.pekko
+import org.apache.pekko.actor.typed.delivery.ConsumerController
```

Comment:
can you remove the org.apache from this import and the next one

Reply:
fixed